index.bs

```diff
@@ -2073,12 +2073,17 @@ Methods</h4>
 
 			1. Abort these steps and reject <var>promise</var> with
 				{{InvalidStateError}} when any of following conditions is true:
-				- The <em>control thread state</em> flag on the {{OfflineAudioContext}}
+				- The <a>control thread state</a> on the {{OfflineAudioContext}}
 					is <code>closed</code>.
 				- The {{[[rendering started]]}} slot on the {{OfflineAudioContext}}
 					is <em>false</em>.
 
-			1. Set the <em>control thread state</em> flag on the
+			If the <a>control thread state</a> flag on the
```

Comment:
This is indented incorrectly somehow.  Wasn't this removed in PR #1887

Reply:
I am not sure how this happened. Removed the paragraph again.